### ISSUE TITLE
`active_is_multidimensional` warnings

### ISSUE THREAD

**Issue by thomasaarholt** [hyperspy/hyperspy#1389]
```python
WARNING:hyperspy.component:`active_is_multidimensional` already False for PowerLaw
```python

Get spit out a lot - reproducable for slicing models by `m.isig[]`. Is there a particular reason for the warning?

**Comment by thomasaarholt** [comment]
Minimum working example:
```python
%matplotlib qt4
import hyperspy.api as hs
n = 4 # Warnings only appear for n > 3… Why?
eds = hs.datasets.example_signals.EDS_TEM_Spectrum()
eds = hs.stack(n*[eds])
m = eds.create_model()

a = m.as_signal()
```

As commented, warnings only appear for n greater than 3.

**Comment by ericpre** [comment]
Fixed in #1492.
